easybuild/easyconfigs/l/LAME/LAME-3.100-GCCcore-13.3.0.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'LAME'
+version = '3.100'
+
+homepage = 'http://lame.sourceforge.net/'
+description = """LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://sourceforge.net/projects/lame/files/lame/%(version_major_minor)s/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['LAME-3.99.5_check-tgetent.patch']
+checksums = [
+    'ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e',  # lame-3.100.tar.gz
+    '8bfb6a73f2db1511baf90fbd7174f11043ec4b592a4917edc30ccfb53bf37256',  # LAME-3.99.5_check-tgetent.patch
+]
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Autotools', '20231222'),
+]
+
+dependencies = [('ncurses', '6.5')]
+
+preconfigopts = "autoconf && "
+
+# configure is broken: add workaround to find libncurses...
+configure_cmd_prefix = "FRONTEND_LDADD='-L${EBROOTNCURSES}/lib' "
```

nitpicking here, but seems more appropriate to add it to preconfigopts?
```suggestion
preconfigopts += "FRONTEND_LDADD='-L${EBROOTNCURSES}/lib' "
```